### PR TITLE
Modulesync 0.6.2 + Release 2.0.0

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ Most useful available parameters:
 
 ## Reference
 
-###Classes 
+###Classes
```
Comment (igalic) [inline README.md]: please add a space between `#*` and the headline
this makes the markdown more… platform… independent…

File: metadata.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "puppet-mysql_java_connector",
-  "version": "1.0.0",
-  "author": "Merritt Krakowitzer",
```
Comment (igalic) [inline metadata.json:4]: perhaps a comment in the changelog? contributors? readme??
that this module was originally written by Merritt

Comment (bastelfreak) [inline metadata.json:4]: added a note in the changelog, like we mostly do.

**Comment by juniorsysadmin** [comment]
I would prefer a Modulesync in one PR, Release change in another. Doing more than one thing in a PR is not a good habit to get into...

**Comment by jyaworski** [comment]
@juniorsysadmin as long as they're separate commits, I think it's fine.

**Pull request merged**